nixie/cli.py

```
@@ -200,28 +203,41 @@ async def render_block(
+            cli,
+        )
+    except RuntimeError:
+        logging.getLogger(__name__).exception(
+            "Runtime error while rendering diagram"
         )
-    except RuntimeError as exc:
-        print(exc, file=sys.stderr)
     except Exception as exc:
-        print(f"{path}: unexpected error in diagram {idx}", file=sys.stderr)
-        traceback.print_exception(type(exc), exc, exc.__traceback__, file=sys.stderr)
+        if isinstance(exc, (KeyboardInterrupt, SystemExit)):
```

Ruff (0.12.2)

Use `X | Y` in `isinstance` call instead of `(X, Y)`

Convert to `X | Y`

(UP038)